src/bun.js/api/server.zig

```diff
@@ -422,6 +491,7 @@ pub const ServerConfig = struct {
                         result.key = native_array;
                         result.key_count = 1;
                         any = true;
+                        result.requires_custom_request_ctx = true;
```

Comment:
is there a case that `any == false` and `requires_custom_request_ctx == true`? dont looks like `requires_custom_request_ctx` will be needed because will return null when any is false

Reply:
No but there are the cases where `any = true` and `requires_custom_context = false`. So it's still needed to determine if the request requires a custom ctx?

I might have misunderstood you question though 😅